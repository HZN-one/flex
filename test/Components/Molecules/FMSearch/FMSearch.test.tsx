/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FMSearch } from "@Components";

import { render } from "@testing-library/react";

describe("FMSearch", () => {
  it("should match snapshot", () => {
    const { container } = render(<FMSearch testID="search-example" />);
    expect(container).toMatchSnapshot();
  });
});
