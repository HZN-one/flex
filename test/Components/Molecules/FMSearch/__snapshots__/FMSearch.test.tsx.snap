// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMSearch should match snapshot 1`] = `
<div>
  <div
    class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
  >
    <div
      class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedStart css-1q6at85-MuiInputBase-root-MuiOutlinedInput-root"
    >
      <div
        class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-ittuaa-MuiInputAdornment-root"
      >
        <span
          class="notranslate"
        >
          ​
        </span>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-colorSecondary MuiSvgIcon-fontSizeMedium css-kw0y8f-MuiSvgIcon-root"
          data-testid="SearchIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
          />
        </svg>
      </div>
      <input
        aria-invalid="false"
        class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedStart css-1o9s3wi-MuiInputBase-input-MuiOutlinedInput-input"
        placeholder="Search"
        type="text"
        value=""
      />
      <fieldset
        aria-hidden="true"
        class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="css-nnbavb"
        >
          <span
            class="notranslate"
          >
            ​
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
